2-Browsers/Week1/homework/ex2-aboutMe/index.js

```diff
@@ -8,4 +8,24 @@
    `list-item`.
 ------------------------------------------------------------------------------*/
 
-// TODO add your JavaScript code here.
+document.body.style.fontFamily = 'Arial, sans-serif';
+
+const nickname = document.getElementById('nickname');
```

Comment:
i think here you can just do nickname.innerText = 'Veronika', no need for all this additional logic